test/stdlib/Math.test.js

```diff
@@ -0,0 +1,66 @@
+const { Exp, Log, Sqr, Rnd } = require("../../lib/stdlib/index");
+const { Interpreter } = require("../../lib/interpreter");
+const { Int32, Float, ValueKind } = require("../../lib/brsTypes");
+
+const interpreter = new Interpreter();
+
+describe("global math functions", () => {
+    describe("Exp", () => {
+        it("calculates a natural exponent", () => {
+            expect(
+                Exp.call(interpreter, new Float(12.7)).value
+            ).toBeCloseTo(327747.901874, 1);
+        });
+    });
+
+    describe("Log", () => {
+        it("calculates a natural logarithm", () => {
+            expect(
+                Log.call(interpreter, new Float(17)).value
+            ).toBeCloseTo(2.833213, 5);
+        });
+    });
+
+    describe("Sqr", () => {
+        it("calculates the square root", () => {
+            expect(
+                Sqr.call(interpreter, new Float(9.9)).value
+            ).toBeCloseTo(3.146426, 5);
+        });
+    });
+
+    describe("Rnd", () => {
+        it("calculates a random float between zero and one", () => {
+            let result = Rnd.call(interpreter, new Int32(0));
+            expect(result.kind).toBe(ValueKind.Float);
+            expect(result.value).toBeGreaterThanOrEqual(0.0);
+            expect(result.value).toBeLessThan(1.0);
+        });
+
+        it("calculates a random number between one and some number", () => {
+            function mockRandomToAlwaysBe(val) {
```

Comment:
Watch out - looks like anything that uses `Math.random` after this function is called will still use the last-mocked value! I've got a commit that fixes that though, so I'll push that in a sec.

Reply:
I figured I wouldn't use random anywhere else in this file and that jest supposedly resets global state per test file, but that's cleaner anyway.